python/pyproject.toml

```diff
@@ -25,6 +25,7 @@ dependencies = [
     "requests",
     "pandas",
     "typing-extensions",
+    "numpy<2",
```

Comment:
Curiosity: which features of NumPy 2+ are non-compatible? As I see it has been recently released.

Reply:
Yeah, we get an error with numpy 2+. 

Seems to be a common thing:
https://stackoverflow.com/questions/78650222/valueerror-numpy-dtype-size-changed-may-indicate-binary-incompatibility-expec